test/e2e/page-objects/pages/confirmations/redesign/batch-confirmation.ts

```diff
@@ -40,9 +48,11 @@ export default class Eip7702AndSendCalls extends TransactionConfirmation {
   }
 
   async tickSplashUpgradeButton(): Promise<void> {
```

Comment:
Minor, should we rename these as it's now a button rather than a checkbox?

Reply:
yeah :heart_eyes:  Nice call, I've updated the button names :pray: